src/components/TaggablePhoto/TaggablePhoto.jsx

```diff
@@ -12,13 +13,23 @@ export default class TaggablePhoto extends React.Component {
   }
 
   async componentDidMount() {
-    // const { taggedUsers } = await fetch(INSERT API CALL FOR PHOTO'S TAGGED USERS HERE);
-    const value = (taggedUsers || []).map(({ email }) => email);
-    this.setState({ value });
+    const {
+      match: {
+        params: { albumId, photoId },
+      },
+    } = this.props;
+    try {
+      const taggedUsers = await fetchTaggedUsers(albumId, photoId);
+      const value = taggedUsers;
+      this.setState({ value });
+    } catch (err) {
+      console.error(err);
+    }
+    
   }
 
   async updateValue(value) {
```

Comment:
Looks like we don't need this function since it's just calling another function. Can just call updateTaggedUsers in hangleChange.